code/modules/vehicles/powerloader.dm

```diff
@@ -95,28 +88,30 @@
 /obj/vehicle/powerloader/post_buckle_mob(mob/buckling_mob)
 	. = ..()
 	playsound(loc, 'sound/mecha/powerloader_buckle.ogg', 25)
-	icon_state = "powerloader"
-	overlays += image(icon_state= "powerloader_overlay", layer = MOB_LAYER + 0.1)
+	icon_state = background_icon
+	overlays += image(icon_state = overlay_icon, layer = MOB_LAYER + 0.1)
 	move_delay = max(4, move_delay - buckling_mob.skills.getRating("powerloader"))
-	var/clamp_equipped = 0
-	for(var/obj/item/powerloader_clamp/PC in contents)
-		if(!buckling_mob.put_in_hands(PC))
-			PC.forceMove(src)
+	var/equipment_equipped = 0
+	var/equipment_required = 0
+	for(var/obj/item/hand in list(hand_one, hand_two))
```

Comment:
Why the typecheck in this list

Reply:
why not

Reply:
Because it's slower and implies the list isn't a type